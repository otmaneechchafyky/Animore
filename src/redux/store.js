import { configureStore } from '@reduxjs/toolkit';
import animeReducer from './anime/AnimeSlice';

const store = configureStore({
  reducer: {
    anime: animeReducer,
  },
});

export default store;
