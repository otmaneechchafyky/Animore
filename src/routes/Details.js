import React from 'react';

const Categories = () => (
  <div>
    okay ..
  </div>
);

export default Categories;
