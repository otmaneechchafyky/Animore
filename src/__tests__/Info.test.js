import React from 'react';
import { render } from '@testing-library/react';
import { Provider } from 'react-redux';
import store from '../redux/store';
import Details from '../components/Info';

test('renders Details component correctly', () => {
  const { container } = render(
    <Provider store={store}>
      <Details />
    </Provider>,
  );
  expect(container.firstChild).toMatchSnapshot();
});
