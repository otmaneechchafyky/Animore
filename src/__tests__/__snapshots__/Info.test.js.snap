// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders Details component correctly 1`] = `
<div>
  Loading...
</div>
`;
